footer.php

```diff
@@ -1,35 +1,35 @@
-
-      <?php wp_footer(); ?>
+    <?php wp_footer(); ?>
 
     <footer class="footer">
-    	<ul>
-			<li class="footer-icons">
-				<a href="https://www.facebook.com">
-					<img src="<?php echo get_stylesheet_directory_uri(); ?>/images/setupshop_ftfacebook1.png" />
-				</a>
-			</li>
-			<li class="footer-icons">
-				<a href="https://www.twitter.com">
-					<img src="<?php echo get_stylesheet_directory_uri(); ?>/images/setupshop_fttwitter1.png" />
-				</a>
-			</li>
-			<li class="footer-icons">
-				<a href="https://www.linkedin.com">
-					<img src="<?php echo get_stylesheet_directory_uri(); ?>/images/setupshop_ftlinkedin.ico" />
-				</a>
-			</li>
-			<li class="footer-icons">
-				<a href="https://www.youtube.com">
-					<img src="<?php echo get_stylesheet_directory_uri(); ?>/images/setupshop_ftyoutube1.png" />
-				</a>
-			</li>
-			<li class="footer-icons">
-				<a href="https://www.instagram.com">
-					<img src="<?php echo get_stylesheet_directory_uri(); ?>/images/setupshop_ftinstagram1.png" />
-				</a>
-			</li>
+      <ul>
+			  <li class="footer-icons">
```

Comment:
Indentation seems bigger from the ul to the li than from the li to the a. Seems like you have a double indent before the li